src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/ContactHistoryService.kt

```diff
@@ -9,6 +9,7 @@ import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecis
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.ContactSummaryResponse
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.ndelius.ReleaseSummaryResponse
 import uk.gov.justice.digital.hmpps.makerecalldecisionapi.reader.ContactGroupsCsvReader
+import kotlin.streams.toList
```

Comment:
Do we need these imports? There are no other changes to that file

Reply:
My IDE complains and errors when it is not there :(

Reply:
Removed